src/ipa/rpi/common/ipa_base.cpp

```diff
@@ -505,7 +512,9 @@ void IpaBase::processStats(const ProcessParams &params)
 			setDelayedControls.emit(ctrls, ipaContext);
 			setCameraTimeoutValue();
 		}
-	}
+	} else
+		LOG(IPARPI, Error) << "Not Running process: pending : " << processPending_;
```

Comment:
I think let's remove it. I'm guessing it was to do with the algorithm getting skipped because of the framerate, but I think we're quite robust to that now.